### PR TITLE
Issue-2-view-entries

### PR DESCRIPTION
**Pull request by hannahshamb**
code-journal-feature-2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -36,4 +41,71 @@ function handleInputs(event) {
   data.entries.push(objectOfValues);
   $displayedImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+
+  $dataViewEntries.classList.remove('hidden');
+  $dataViewEntryForm.classList.add('hidden');
+  data.view = 'entries';
+  location.reload();
```
Comment (BrettAlbright) [inline js/main.js]: you want to avoid reloading the app during run time.

File: js/main.js [hunk 2]
```diff
@@ -36,4 +41,71 @@ function handleInputs(event) {
...
 }
+
+window.addEventListener('DOMContentLoaded', addAnEntry);
+function addAnEntry(entry) {
+  var ulListEntries = document.createElement('ul');
+  ulListEntries.setAttribute('class', 'list-entries');
+  if (data.entries.length > 0) {
+    for (var i = 0; i < data.entries.length; i++) {
+      var liRowListItem = document.createElement('li');
```
Comment (BrettAlbright) [inline js/main.js]: so one thing you want to do is have the function to render the `lis` as a separate render function, like in the javascript dom creation exercise. The loop should call the function, pass in the entry, then append the returning entry. you want to make the all the function does is create the DOM tree and return it, you don't want to append it in the function.

File: js/main.js (hunk 1 above)
Comment (BrettAlbright) [inline js/main.js]: if the goal is to render the entries in real time, the best approach will be to call the function to render the entries, then use the prepend method to put the entry as the first child on the DOM in the ul.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great! Just a couple functionality updates

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**